apiary.apib

```diff
@@ -4,7 +4,107 @@ FORMAT: 1A
 
 The API for interacting with IPFS nodes.
 
-# Group add
+# Group add [POST /add{?arg}{&recursive,quiet,silent,progress,trickle,only%2Dhash,wrap%2Dwith%2Ddirectory,hidden,chunker,pin}]
```

Comment:
agree that `add` should be locked to be only a POST method

I believe the API is only expecting the aliases for the flags though (recursive -> r, quiet -> q, etc)